fdbrpc/Net2FileSystem.cpp

```diff
@@ -93,6 +93,9 @@ Net2FileSystem::Net2FileSystem(double ioTimeout, std::string fileSystemPath)
 {
 	Net2AsyncFile::init();
 #ifdef __linux__
+	if (FLOW_KNOBS->DISABLE_POSIX_KERNEL_AIO)
+		return;
+
 	AsyncFileKAIO::init( Reference<IEventFD>(N2::ASIOReactor::getEventFD()), ioTimeout );
 
 	if (fileSystemPath.empty()) {
```

Comment:
Do we need to skip this check if KAIO is disabled?

Reply:
The existing code always calls `AsyncFileKAIO::init` on Linux. However, the issue is that Windows Subsystem for Linux doesn't implement KAIO but is Linux build, so we always have to do this check to avoid calling `init and crashing.

Reply:
Sorry, I mean the check at line 101 and below that is validating some property of mount points.

Reply:
Yeah, you are right. WSL can run these checks. Fixed it in my latest push.